scripts/docs_version_pragma_check.sh

```diff
@@ -32,13 +32,54 @@ SOLIDITY_BUILD_DIR=${SOLIDITY_BUILD_DIR:-build}
 source "${REPO_ROOT}/scripts/common.sh"
 source "${REPO_ROOT}/scripts/common_cmdline.sh"
 
+function versionGreater() {
```

Comment:
Please use a similar style as we use in C++: `{`, `then` and `do` on the next line